config/sync/core.entity_form_display.node.vba_facility.default.yml

```
@@ -110,8 +119,10 @@ third_party_settings:
         - field_address
         - field_geolocation
         - field_phone_number
+        - group_benefits_hotline
         - field_office_hours
         - field_timezone
+        - group_benefit_office_hours_may_v
```

I think I am with Dave's suggestion on this 
```suggestion
        - group_benefit_office_hours_vary
```

---

I must have missed this. Done.